cszczepaniak/cribbly-backend
# Add game classes

# What I added (link any issues addressed)
All classes that should be needed pertaining to the gameplay itself

### Classes
- Player
- Team
- BracketGame and PlayInGame (Implement IGame)
- Division
- Tournament

### Logic

- TournamentController (Only returns sample data for now)

# How to test it
Run the app and navigate to http://localhost:5000/tournament using curl or a browser. A blank JSON structure should be returned
